internal/providersdkv2/data_source_hvn_route.go

```diff
@@ -58,6 +58,26 @@ Remove this attribute from the configuration for any affected resources.
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"azure_config": {
+				Description: "The azure configuration for routing.",
+				Type:        schema.TypeList,
+				Computed:    true,
+				Optional:    true,
```

Comment:
Should this have validation that a single element is allowed? `MaxItems: 1,`

Reply:
Ah good point, that makes sense. Considering further, there is likely no value to having this be an input at all right? Should we just remove the `Optional: true`?

Reply:
Ah sorry, didn't mean to put the comment on the data_source, but the resource. So ignore this.

Reply:
Perfect, ok tnx